modules/react/breadcrumbs/stories/examples/Overflow.tsx

```diff
@@ -21,11 +21,16 @@ export const OverflowBreadcrumbs = () => {
   ]);
 
   const model = useBreadcrumbsModel({items});
-  const [containerWidth, setContainerWidth] = React.useState('350px');
-
+  const [containerWidth, setContainerWidth] = React.useState(width);
+  console.log('width', width);
   return (
     <div>
       <Box width={containerWidth} marginBottom="xl">
+        <div style={{display: 'flex', flexDirection: 'column'}}>
+          <span>Current Container Width: {containerWidth}</span>
+          <span>Overflow visibility: {model.menu.state.visibility}</span>
```

Comment:
Maybe we don't need this anymore?

Reply:
I don't mind keeping this around. 

It at least gives us the visual